packages.json

```diff
@@ -2680,6 +2680,7 @@
   "https://github.com/finestructure/Plotty.git",
   "https://github.com/finestructure/Rester.git",
   "https://github.com/finestructure/SemanticVersion.git",
+  "https://github.com/finestructure/swift-package-manager.git",
```

Comment:
This is coming in via our dependency on the fork in SPI-Server. I'll merge this and then remove it again via the deny-list to keep it out.